client/src/util/pages.jsx

```diff
@@ -77,6 +79,18 @@ export const pages = {
       path: '/profile',
       includeFooter: true,
     },
+    {
+      label: 'Unsubscribe',
+      component: <PageUnsubscribe />,
+      path: '/unsubscribe',
+      includeFooter: false,
+    },
+    {
+      label: 'Resubscribe',
+      component: <PageResubscribe />,
+      path: '/resubscribe',
+      includeFooter: false,
+    },
```

Comment:
Can you make this page require the user to be logged in? Also, if you could add a button on the profile page to redirect to here that would be amazing!

Reply:
The button should only be displayed if `canEmail` is false